parser/parser.y

```diff
@@ -560,6 +561,15 @@ AlterSpecification:
 KeyOrIndex:
 	"KEY"|"INDEX"
 
+ColumnOptColumnName:
+	"COLUMN"
+|	"COLUMN" "COLUMN"
```

Comment:
In this way I reduce S/R conflict from 25 to 24. Is this a better way? @cznic 
diff --git a/parser/parser.y b/parser/parser.y
index 8b70e02..96b6586 100644
--- a/parser/parser.y
+++ b/parser/parser.y
@@ -531,7 +531,14 @@ AlterSpecification:
                        Constraint: constraint,
                }
        }
-|      "DROP" ColumnKeywordOpt ColumnName
+|      "DROP" ColumnName
+       {
+               $$ = &ddl.AlterSpecification{
+                       Action: ddl.AlterDropColumn,
+                       Name: $2.(string),
+               }
+       }
+|      "DROP"  "COLUMN" ColumnName
        {
                $$ = &ddl.AlterSpecification{
                        Action: ddl.AlterDropColumn,